CMakeLists.txt

```diff
@@ -34,6 +34,9 @@ set(IREE_IDE_FOLDER IREE)
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
 option(IREE_ENABLE_RUNTIME_TRACING "Enables instrumented runtime tracing." OFF)
+if(${IREE_ENABLE_RUNTIME_TRACING})
+  set (CMAKE_EXE_LINKER_FLAGS -ldl )
```

Comment:
weird spaces here
can you instead move this to iree_copts so that we have a single place for these kind of settings?

Reply:
I move it to build_tools/cmake/iree_copts.cmake, is it a right place?